code/modules/buildmode/buttons.dm

```diff
@@ -1,7 +1,7 @@
 /obj/screen/buildmode
 	icon = 'icons/misc/buildmode.dmi'
 	var/datum/click_intercept/buildmode/bd
-	layer = HUD_LAYER_BUILDMODE
+	plane = HUD_PLANE_BUILDMODE
```

Comment:
Have you tested with just `plane = ABOVE_HUD_PLANE`

I would rather use that than yet another plane.

Reply:
`ABOVE_HUD_PLANE` works, but not flawlessly:
![Paradise Test NSS Cyberiad 2021-08-15 142355](https://user-images.githubusercontent.com/57483089/129480087-1506ed54-bcae-4f2d-9bf3-37173a01feb7.png)
It's rendering above the darkening overlay since that's set to `FLOAT_PLANE + 1` (22), but the cooldown text is `FLOAT_PLANE +1.1` (22.1) so it's still taking priority. (`#define ABOVE_HUD_PLANE 22`)
This still fixes the main issue, but build mode being on its own plane fixes everything else too and is probably better for future-proofing.

Reply:
Please put the buildmode plane in the file with the rest of the planes then